pkg/genericapiserver/config.go

```diff
@@ -394,21 +402,34 @@ func (c Config) New() (*GenericAPIServer, error) {
 		handler = authenticatedHandler
 	}
 
-	// TODO: Make this optional?  Consumers of GenericAPIServer depend on this currently.
-	s.Handler = handler
-
-	// After all wrapping is done, put a context filter around both handlers
-	var err error
-	handler, err = api.NewRequestContextFilter(c.RequestContextMapper, s.Handler)
+	handler, err := api.NewRequestContextFilter(c.RequestContextMapper, handler)
 	if err != nil {
 		glog.Fatalf("Could not initialize request context filter for s.Handler: %v", err)
 	}
+
+	longRunningRE := regexp.MustCompile(c.LongRunningRequestRE)
```

Comment:
This is part of the follow-up PR https://github.com/kubernetes/kubernetes/pull/33164.